content/reference/host-runtime/capabilities.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Capabilities"
+date: 2018-12-29T11:02:05+06:00
+weight: 5
+draft: false
+---
+
+A **_capability_** is an abstraction or representation of a non-functional requirement; some functionality required by your actor that is not considered part of the core business logic. As you write an actor that exposes some data over a RESTful endpoint, the HTTP server is not part of your business logic, it is a service used by your actor--a capability.
+
+In wasmCloud, _capability providers_ are dynamic libraries that implement a _capability contract_. A capability contract is a unique name that identifies the interface or abstraction. By convention, these capability contract IDs are prefixed by a vendor ID (the vendor of the _contract_, not necessarily the specific _implementation_). For example, the following is a list of the first-party capability contract IDs supported by default wasmCloud providers:
+
+* wasmcloud:http_server
+* wasmcloud:http_client
```

Comment:
In my actor interfaces PR, the contract is `wasmcloud:httpserver`. I was under the impression we decided to go no-underscores because of the difference between `keyvalue` and `http_server`.

Reply:
Good point. Documentation-driven development FTW.